ci-operator/step-registry/aws/provision/bastionhost/aws-provision-bastionhost-commands.sh

```diff
@@ -75,10 +75,19 @@ if [[ "${BASTION_HOST_AMI}" == "" ]]; then
   fi
   #Use 4.18 RHCOS image as the boot image of bastion by default
   bastion_image_list_url="https://raw.githubusercontent.com/openshift/installer/release-4.18/data/data/coreos/rhcos.json"
-  curl -sL "${bastion_image_list_url}" -o /tmp/bastion-image.json
+  if ! curl -sSLf --retry 3 --connect-timeout 30 --max-time 60 -o /tmp/bastion-image.json "${bastion_image_list_url}"; then
```

Comment:
This reminds me a problem we hit before when downloading `yq` from github, sounds like there is a limit when too much downloading happening in a short time. 

How about copying the required aws AMI info, create a temp json file in this step to avoid downloading the whole rhcos.json

Reply:
@jianlinliu we're using the URL is because we found that the old images in certain regions may be pruned somehow, so we changed the reference link to point to release-4.18 directly.